00_jit.ipynb

```diff
@@ -53,180 +63,56 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "#exporti\n",
-    "@patch\n",
-    "def requires_grad_(self:TensorBase, requires_grad=True):\n",
-    "    # Workaround https://github.com/pytorch/pytorch/issues/50219\n",
-    "    self.requires_grad = requires_grad\n",
-    "    return self"
+    "# #exporti\n",
+    "# @patch\n",
+    "# def requires_grad_(self:TensorBase, requires_grad=True):\n",
+    "#     # Workaround https://github.com/pytorch/pytorch/issues/50219\n",
+    "#     self.requires_grad = requires_grad\n",
+    "#     return self"
    ]
   },
   {
-   "cell_type": "code",
-   "execution_count": null,
+   "cell_type": "markdown",
    "metadata": {},
-   "outputs": [],
    "source": [
-    "#slow\n",
-    "from fastai.vision.all import *"
+    "There are two possible scenarios with `jit`: `trace` and `script`. \n",
+    "\n",
+    "**I don't think there is an ideal** Ideally you should use `torch.jit.trace`, however if there is dynamic behavior, `torch.jit.script` should be utilized instead. As a result `trace` is tried by default."
```

Comment:
Phrasing, You say "I don't think there's an ideal" and then say "Ideally".

Reply:
lol this was just to comment that the phrase :Ideally you should use" is not correct